.buildkite/scripts/backport_branch.sh

```diff
@@ -0,0 +1,177 @@
+    return 0
+  else
+    echo "The backport branch $branch does not exist"
+    return 1
+  fi
+}
+
+createLocalBackportBranch() {
+  local branch_name=$1
+  local source_commit=$2
+  if git checkout -b "$branch_name" "$source_commit"; then
+    echo "The branch $branch_name has created."
+  else
+    buildkite-agent annotate "The backport branch **$BACKPORT_BRANCH_NAME** hasn't created." --style "warning"
+    exit 1
+  fi
+}
+
+removeOtherPackages() {
+  local sourceFolder=$1
+  for dir in "$sourceFolder"/*; do
+    if [[ -d "$dir" ]] && [[ "$(basename "$dir")" != "$PACKAGE_NAME" ]]; then
+      echo "Removing directory: $dir"
+      rm -rf "$dir"
+    fi
+  done
+}
+
+updateBackportBranch() {
+  local BUILDKITE_FOLDER_PATH=".buildkite"
+  local JENKINS_FOLDER_PATH=".ci"
+  git checkout $BACKPORT_BRANCH_NAME
+  echo "Copying $BUILDKITE_FOLDER_PATH from $SOURCE_BRANCH..."
+  git checkout $SOURCE_BRANCH -- $BUILDKITE_FOLDER_PATH
+  echo "Copying $JENKINS_FOLDER_PATH from $SOURCE_BRANCH..."
+  git checkout $SOURCE_BRANCH -- $JENKINS_FOLDER_PATH
+
+  if [ "${REMOVE_OTHER_PACKAGES}" == "true" ]; then
+    echo "Removing all packages from $PACKAGES_FOLDER_PATH folder"
+    removeOtherPackages "$PACKAGES_FOLDER_PATH"
+    ls -la $PACKAGES_FOLDER_PATH
+  fi
+
+  echo "Setting up git environment..."
+  git config --global user.name "${GITHUB_USERNAME_SECRET}"
+  git config --global user.email "${GITHUB_EMAIL_SECRET}"
+
+  echo "Commiting and pushing..."
+  git add $BUILDKITE_FOLDER_PATH
+  git add $JENKINS_FOLDER_PATH
+  git commit -m "Add $BUILDKITE_FOLDER_PATH and $JENKINS_FOLDER_PATH to backport branch: $BACKPORT_BRANCH_NAME from the $SOURCE_BRANCH branch"
+  git push origin $BACKPORT_BRANCH_NAME
+}
+
+if ! [[ $PACKAGE_VERSION =~ ^[0-9]+(\.[0-9]+){2}$ ]]; then
+  buildkite-agent annotate "The entered package version ${PACKAGE_VERSION} doesn't match the pattern: X.Y.Z" --style "error"
+  exit 1
+fi
+
+add_bin_path
+
+with_yq
+
+echo "Check the entered version and PACKAGE_VERSION are equal"
+version="$(cat packages/${PACKAGE_NAME}/manifest.yml | yq -r .version)"
+if [[ "${version}" != "${PACKAGE_VERSION}" ]]; then
+  buildkite-agent annotate "Unexpected version found in packages/${PACKAGE_NAME}/manifest.yml" --style "error"
+  exit 1
+fi
+
+echo "Check that this changeset is the one creating the version $PACKAGE_NAME"
+if ! git show -p ${BASE_COMMIT} packages/${PACKAGE_NAME}/manifest.yml | grep -E "^\+version: ${PACKAGE_VERSION}" ; then
+  buildkite-agent annotate "This changeset does not creates the version ${PACKAGE_VERSION}" --style "error"
+  exit 1
+fi
+
+echo "Check if the package has published"
+if ! isPackagePublished "$FULL_ZIP_PACKAGE_NAME"; then
+  buildkite-agent annotate "The package version: **${PACKAGE_NAME}-${PACKAGE_VERSION}** hasn't neen published yet." --style "error"
+  exit 1
+fi
+
+echo "Check if the base commit exists."
+if [ ! -z "$BASE_COMMIT" ]; then
+  if ! isCommitExist "$BASE_COMMIT" "$SOURCE_BRANCH"; then
```

Comment:
Nit. I would rename this function

```suggestion
  if ! doesCommitExist "$BASE_COMMIT" "$SOURCE_BRANCH"; then
```

or 
```suggestion
  if ! commitExists "$BASE_COMMIT" "$SOURCE_BRANCH"; then
```

or 
```suggestion
  if ! commitPartofBranch "$BASE_COMMIT" "$SOURCE_BRANCH"; then
```

Reply:
changed